.changeset/afraid-mangos-sip.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-kubernetes': minor
```

Comment:
Are there any breaking changes? Otherwise this should be `patch` since these packages are at 0.x